code/modules/mob/living/simple_animal/abnormality/_tools/teth/skin.dm

```diff
@@ -7,32 +7,44 @@
 
 /obj/structure/toolabnormality/skin/attack_hand(mob/living/carbon/human/user)
 	..()
-	if(do_after(user, 6))
-		if(get_level_buff(user, PRUDENCE_ATTRIBUTE) >= 100)
-			to_chat(user, "<span class='userdanger'>You've learned all that you could.</span>")
-			return	//You don't need any more.
-		user.adjust_attribute_buff(PRUDENCE_ATTRIBUTE, 10)
-
-		if(!(user in readers))
-			readers+= user
-		else
-			user.physiology.white_mod *= 1.15
-
-		user.apply_status_effect(STATUS_EFFECT_SKIN)
-		to_chat(user, "<span class='userdanger'>You read the book, and take the time to burn these passages into your brain.</span>")
-
-//SKIN
-//this keeps track of dying
+	if(!do_after(user, 6))
+		return
+	if(get_level_buff(user, PRUDENCE_ATTRIBUTE) >= 100)
+		to_chat(user, "<span class='notice'>You've learned all that you could.</span>")
+		return //You don't need any more.
+
+	user.adjust_attribute_buff(PRUDENCE_ATTRIBUTE, 10)
+	if(!(user in readers))
+		readers += user
+	else
+		user.physiology.white_mod *= 1.15
+
+	user.apply_status_effect(STATUS_EFFECT_SKIN)
+	to_chat(user, "<span class='userdanger'>You read the book, and take the time to burn these passages into your brain.</span>")
+
+// Status Effect
 /datum/status_effect/skin
 	id = "skin"
 	status_type = STATUS_EFFECT_UNIQUE
 	duration = -1
 	alert_type = null
 
-/datum/status_effect/skin/tick()
+/datum/status_effect/behavior/on_apply()
```

Comment:
I believe this is a mistype and should be `/skin/on_apply()`

Reply:
Oh right, I was just copy-pasting what I wrote for behavior.